index.html

```diff
@@ -54,7 +56,7 @@
                 </nav>
             </header>
             
-            <div class="content to-be-section">
+            <div class="content to-be-section-b5 section">
```

Comment:
- [ Optional ] I think it would be better if you made sure that the `background image` in the hero section of the `index.html` and `about.html` pages are also visible on mobile devices. Currently, it appears that the background images are only visible on screen sizes above `768px`. Please see the screenshots below for more details on this. I highly recommend you implement this for your design to be consistent with the design on Behance.net

## Your design
|Home page | About page|
|----------------|----------------|
|<img src="https://github.com/Paulinelali/Microverse-capstone-one/assets/16392046/0e4350d6-d6ab-4619-99bd-2de675b0db92" width="375px" />| <img src="https://github.com/Paulinelali/Microverse-capstone-one/assets/16392046/2b8ce027-a77a-4de2-987c-3c0fd33a4544" width="375px" />|



## Behance design
![Screenshot from 2023-05-25 02-37-33](https://github.com/Paulinelali/Microverse-capstone-one/assets/16392046/2433421e-fd0b-4c7d-b93e-a12f73c6fbb0)